src/ndb_p1_section.c

```diff
@@ -0,0 +1,183 @@
+/*
+ * Copyright (C) 2016 Swift Navigation Inc.
+ * Contact: Roman Gezikov <rgezikov@exafore.com>
+ *
+ * This source is subject to the license found in the file 'LICENSE' which must
+ * be be distributed together with this source. All other rights reserved.
+ *
+ * THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND,
+ * EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
+ */
+
+
+#include "ndb_p1_section.h"
+
+#include <string.h>
+#include "ndb.h"
+
+#define GPS_L1_EPHE_FILE_NAME "gps_l1_ephe"
```

Comment:
I guess it's a matter of preference. I cannot mathematically prove that but my intuition says that keeping things separate makes sense. However, I prepared a version of code with NDB_COMMON_STORAGE #ifdef (temporary one, I'll delete it). Please review and let me know which branch to delete.